packages/@react-spectrum/datepicker/stories/DateRangePicker.stories.tsx

```diff
@@ -78,6 +78,12 @@ HourCycle24.story = {
   name: 'hourCycle: 24'
 };
 
+export const ForceLeadingZeros = () => render({defaultValue: {start: new CalendarDate(2020, 2, 3), end: new CalendarDate(2020, 5, 4)}, shouldForceLeadingZeros: true});
```

Comment:
Do we need a new story when it's already in the controls?

Reply:
DateRangePicker and TimeField don't have controls for some reason. I added to the controls for DatePicker and DateField. Seems like we need to convert the others over at some point.